integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -90,7 +90,14 @@ def __init__(
         self._port = port
         self._chroma_client = None
 
-    @property
+        self._initialized = False
+
+    def _ensure_initialized(self):
+        if not self._initialized:
+            self._chroma_client = self.chroma_client()
+            self._collection = self.collection()
+            self._initialized = True
```

Comment:
Let's inline these two function calls and remove the individual methods - I don't think they are used anywhere else.